frontend/src/pages/ActivityReport/Pages/Review/ReviewItem.js

```diff
@@ -59,19 +81,36 @@ const ReviewItem = ({
     values.sort();
   }
 
-  values = values.map((v) => {
+  values = values.map((v, index) => {
     // If not a valid url, then its most likely just text, so leave it as is
     // except for several values
     if (!isValidURL(v)) {
+      if (isRichText) {
+        return (
+          <Editor
+            readOnly
+            toolbarHidden
+            defaultEditorState={getEditorState(v)}
+          />
+        );
+      }
       return mapUrlValue(v);
     }
 
+    const linkNameToUse = linkNamePath ? linkNameValues[index] : v;
+    if (isFile) {
+      return (
+        <a href={v} onClick={onHubFileLinkClick}>
```

Comment:
Please convert this to putting target="_blank" on the link

Reply:
We also already have a <FileReviewItem> component which may be useful here

Reply:
What is the component name?

Reply:
I used this to get around the modal warning the user they are going to an external link. By this point we know its a HUB file.